src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import DropdownControl, {DropdownItem} from 'app/components/dropdownControl';
+import Button from 'app/components/button';
+import GroupList from 'app/views/releases/detail/groupList';
+import space from 'app/styles/space';
+import {Panel} from 'app/components/panels';
+import EventView from 'app/views/eventsV2/eventView';
+import {formatVersion} from 'app/utils/formatters';
+
+type Props = {
+  orgId: string;
+  version: string;
+};
+
+const Issues = ({orgId, version}: Props) => {
+  // TODO(releasesV2): figure out the query we want + do we want to pass globalSelectionHeader values?
+  const getDiscoverUrl = () => {
+    const discoverQuery = {
+      id: undefined,
+      version: 2,
+      name: `${t('Release')} ${formatVersion(version)}`,
+      fields: ['title', 'count(id)', 'event.type', 'user', 'last_seen'],
+      query: `release:${version}`,
```

Comment:
Do you only want errors?

Reply:
This query is more like a temporary placeholder just so it links to the discover. I will ask @fdrabek to decide what the final discover result should look like and then I will probably bug you with it a little bit :)